src/game.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const readline = require('readline');
+const { checkForDublicates } = require('./checkForDublicates');
+const { calculateBullsAndCows } = require('./calculateBullsAndCows');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+function game(message, generatedNumber) {
+  terminal.question(message, (guessedNumber) => {
+    if (guessedNumber.length !== 4 || checkForDublicates(guessedNumber)) {
```

Comment:
What is this number actually mean?
Do you remember why it's equal to 4 after a month?
It would help if you created a specific variable for such case